src/chord.js

```diff
@@ -296,7 +297,14 @@ var ChordChart = {
       datamatrix = [];
       //read in data and create chord when finished
       wholeDataTotal = 0;
-      d3.csv(url, function(error, data) {
+
+      if (csvData) {
```

Comment:
We don't need to go back to the server for the csv every time we have to redraw the chart.

Reply:
That's why I added this block, that is.